### DISCUSSION TITLE
解决豆瓣小组页面动态网址的问题

### DISCUSSION THREAD

**Discussion by binsee** [Kenshin/simpread#3206] [站点适配]
## 说明
在 TG 群中有人提到豆瓣小组的页面每次刷新，网址就会变，导致加入稍后读、标注、阅读模式之间识别会出现问题。
通过查看豆瓣小组的页面，发现页面在载入后会修改网址，在网址中加入了包含部分时间戳的锚点。因此每次刷新页面，完整网址都不一样。

## 解决方法

由于简悦自身暂不支持在适配规则与插件中对已提取的页面网址进行修正，因此提供了油猴脚本的解决方案。

1. 安装油猴 [Tampermonkey • 首页](https://www.tampermonkey.net/)
2. 安装油猴脚本 [移除网址中的锚点](https://greasyfork.org/zh-CN/scripts/437102-%E7%A7%BB%E9%99%A4%E7%BD%91%E5%9D%80%E4%B8%AD%E7%9A%84%E9%94%9A%E7%82%B9)

## 效果示例页面
- [是不是要有日更的新节目了？？？？？](https://www.douban.com/group/topic/255233265/#) （来自 TG 群提交的网址）

```
// 原网址
https://www.douban.com/group/topic/255233265/#9579965H5MOWHs
https://www.douban.com/group/topic/255233265/#9580005H5MOWHs

// 插件修正后的网址
https://www.douban.com/group/topic/255233265/#
```

**Comment by binsee** [comment]
如果有发现其他类似情况的站点，可以在此告知，以跟进更新适配。

**Reply by Kenshin** [reply]
嗨，我把这个 discussion 转移到适配规则里面了。
